infrastructure/src/main/resources/reference.conf

```diff
@@ -1,5 +1,5 @@
 influx {
-  url = "http://10.90.36.34:8086"
+  url = "http://10.90.40.6:8086"
```

Comment:
I think this is where it's getting pulled from when we run the benchmarks. So I believe this will fix it.